### PR TITLE
update config-files #14

### PR DESCRIPTION
**Pull request by koilebeit**
# Pull request

## Proposed changes

update sgb specific config (#14)

## Types of changes

- [ ] New feature (non-breaking change which adds functionality).
- [x] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **Chores**
	- Updated the `.gitignore` configuration to allow `objects/` directory to be committed.

### CODE REVIEW

**Review by maehr — CHANGES_REQUESTED** [review]
Thanks a bunch! Please incorporate Omeka fields into `config-table.csv`

**Comment by maehr** [comment]
@koilebeit Thanks for the PR. Can you remove the link to the map from the navigation bar?

**Comment by maehr** [comment]
@koilebeit Can you use the new view you implemented for the subjects?

**Comment by maehr** [comment]
@koilebeit please add nofollow and target=_blank to the links to Zotero and Zenodo

**Comment by koilebeit** [comment]
> @koilebeit Can you use the new view you implemented for the subjects?

This is already covered by the corresponding PR and will be automatically implemented after a merge of this PR

> @koilebeit please add nofollow and target=_blank to the links to Zotero and Zenodo

This would mean a change in includes/collection-nav.html. Shouldn't I rather implement this in a separate PR, as this does not only affect the config files?

**Comment by maehr** [comment]
> > @koilebeit Can you use the new view you implemented for the subjects?
> 
> This is already covered by the corresponding PR and will be automatically implemented after a merge of this PR
> 
> > @koilebeit please add nofollow and target=_blank to the links to Zotero and Zenodo
> 
> This would mean a change in includes/collection-nav.html. Shouldn't I rather implement this in a separate PR, as this does not only affect the config files?

Thank you for reviewing my comments. We should push my requests to separate PRs as you suggested.

File: _data/config-nav.csv [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-Subjects,/subjects.html,
-Locations,/locations.html,
-Map,/map.html,
-Timeline,/timeline.html,
-Data,/data.html,
-About,/about.html,
+Sammlung,/browse.html,
+Themen,/subjects.html,
+Orte,/locations.html,
+Karte,/map.html
```
Comment (maehr) [inline _data/config-nav.csv:6]: Please remove locations and map from the va bar

**Review by maehr — APPROVED** [review]

**Pull request merged**